skymap_scanner/server/start_scan.py

```diff
@@ -126,6 +126,8 @@ def __init__(
 
         self.omgeo = g_frame["I3Geometry"].omgeo
 
+        self.ang_dist, self.online_ra_dec = self.reco.get_pointing_info(p_frame)
```

Comment:
These attributes appear innate (constant) to the reco algo instance. So duplicating these attrs is unnecessary and can eventually lead to some sticky situations. For example, `self.online_ra_dec` is changed here, but `self.reco`'s value doesn't. I think `self.reco.ang_dist` and `self.reco.online_ra_dec` would suffice